tests/units/anta_tests/snmp/data.py

```diff
@@ -0,0 +1,146 @@
+"""Data for testing anta.tests.snmp"""
+
+from typing import Any, Dict, List
+
+INPUT_SNMP_STATUS: List[Dict[str, Any]] = [
+    {
+        "name": "success",
+        "eos_data": [
+            {
+                'vrfs': {'snmpVrfs': ['MGMT', 'default']},
+                'enabled': True
+            }
+        ],
+        "side_effect": "MGMT",
+        "expected_result": "success",
+        "expected_messages": []
+    },
+    {
+        "name": "failure",
+        "eos_data": [
+            {
+                'vrfs': {'snmpVrfs': ['default']},
+                'enabled': True
+            }
+        ],
+        "side_effect": "MGMT",
+        "expected_result": "failure",
+        "expected_messages": ["SNMP agent disabled in vrf MGMT"]
+    },
+    {
+        "name": "failure",
```

Comment:
Use different names for both failures so that it is easy to know which test we are looking at when things failed